# c++/12/bits/basic_string.h in libstdc++-devel-12.1.1-3 no longer includes <ext/atomicity.h>
# atomicity.h includes  gthr -> gthr-default -> pthread .h -> time.h -> struct-tm.h
# so explicitly include time.h

--- testsuite-12.1.0/src/CmdLine.C
+++ testsuite-12.1.0/src/CmdLine.C
@@ -38,5 +38,3 @@
 
-#if defined(os_windows_test)
 #include <time.h>
-#endif
 
